scripts/projects-test/src/performBrowserTest.ts

```diff
@@ -1,54 +1,92 @@
-import http from 'http';
+import { Server } from 'http';
 import { AddressInfo } from 'net';
 
-import { safeLaunchOptions } from '@fluentui/scripts-puppeteer';
-import express from 'express';
-import puppeteer from 'puppeteer';
+import { launch, visitUrl } from '@fluentui/scripts-puppeteer';
+import express, { Express } from 'express';
 
+/**
+ *
+ * .close is asynchronous (not promisified thus we are wrapping it)
+ *  @see https://nodejs.org/api/net.html#serverclosecallback
+ */
+export function closeServer(server: Server): Promise<void> {
+  if (!server) {
+    return Promise.resolve();
+  }
 
-const SERVER_HOST = 'localhost';
+  return new Promise<void>((resolve, reject) => {
+    // TODO: Default is 5 seconds. why is this set to 1 ?
+    // https://nodejs.org/api/http.html#serverkeepalivetimeout
+    server.keepAliveTimeout = 1000;
+    server.close(err => (err ? reject(err) : resolve()));
+  });
+}
 
-function startServer(publicDirectory: string, listenPort: number) {
-  return new Promise<http.Server>((resolve, reject) => {
+export function startServer(
```

Comment:
why not make this function async?

Reply:
there is no need. 
- no `await` used within
- we would need to create manuall promise anyways to make express awaitable